core/src/main/java/com/amplifyframework/auth/AuthException.java

```diff
@@ -60,8 +60,8 @@ public AuthException(
      */
     public static class InvalidStateException extends AuthException {
         private static final long serialVersionUID = 1L;
-        private static final String MESSAGE = "Auth state reached an invalid state.";
-        private static final String RECOVERY_SUGGESTION = "Please reset auth plugin and reattempt the operation.";
+        private static final String MESSAGE = "Auth state is an invalid state, cannot process the request.";
```

Comment:
nit: Are customers aware of state machine? If not, how about just having something like `Auth is in an invalid state` etc? Can we also add the context on *what* the request is? if not possible here, we can define that where the exception is thrown.